### ISSUE TITLE
Runs in 6.02 ms... Nothing actually happens.

### ISSUE THREAD

**Issue by kentcdodds** [shinnn/gulp-gh-pages#29]
For about an hour I banged my head against a wall... When trying to run with this simple configuration:

```javascript
var gulp = require('gulp');
var deploy = require('gulp-gh-pages');
gulp.task('deploy', function () {
  gulp.src('./demo/**/*')
    .pipe(deploy());
});
```

I was getting this:

```
$ gulp deploy
[00:05:02] Using gulpfile ~/Developer/marvel-random/gulpfile.js
[00:05:02] Starting 'deploy'...
[00:05:02] Finished 'deploy' after 6.02 ms
```

As you can see, it finished way to quickly to actually have done anything. Changing to this made it work:

```javascript
var gulp = require('gulp');
var deploy = require('gulp-gh-pages');
gulp.task('deploy', function () {
  gulp.src('./demo/**/*')
    .pipe(deploy())
    .pipe(gulp.dest('.tmp')); // added this line
});
```

I'm not sure what the issue was, but I believe it has to do with the size of the directory I'm uploading (4 MB). Am I doing something wrong?

**Comment by astericky** [comment]
IDK, This doesn't actually work as advertised for me. I would love to use it though.

**Comment by shinnn** [comment]
Released v0.5.0 which includes various fixes. Please update gulp-gh-pages to the latest version and try again.
